daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -699,6 +699,27 @@ class RazerCynosaChroma(_RippleKeyboard):
     DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1256/1256_cynosa_chroma.png"
 
 
+class RazerCynosaChromaV2(_RippleKeyboard):
+    """
+    Class for the Razer Cynosa Chroma V2
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Cynosa_Chroma_V2(-if01)?-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x025E
+    HAS_MATRIX = True
+    WAVE_DIRS = (0, 1)
+    MATRIX_DIMS = [6, 22]
+    METHODS = ['get_device_type_keyboard', 'set_wave_effect', 'set_static_effect', 'set_spectrum_effect',
+               'set_reactive_effect', 'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row', 'get_game_mode', 'set_game_mode', 'get_macro_mode', 'set_macro_mode',
+               'get_macro_effect', 'set_macro_effect', 'get_macros', 'delete_macro', 'add_macro',
+               'set_starlight_random_effect', 'set_starlight_single_effect', 'set_starlight_dual_effect',
+               'set_ripple_effect', 'set_ripple_effect_random_colour']
+
+    DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1256/1256_cynosa_chroma.png"
```

Comment:
It looks like the device image could be updated here: https://assets.razerzone.com/eeimages/support/products/1694/1694-2.png

Reply:
thanks for the suggestions!